src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -58,16 +74,28 @@ def _make_request(self):
-            
+            transaction_get_record.includeDuplicates = self.include_duplicates
             query = query_pb2.Query()
             if not hasattr(query, 'transactionGetRecord'):
                 raise AttributeError("Query object has no attribute 'transactionGetRecord'")
             query.transactionGetRecord.CopyFrom(transaction_get_record)
-            
+    
             return query
         except Exception as e:
-            print(f"Exception in _make_request: {e}")
             raise
```

Ruff (0.14.14)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)